G/GDB/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GDB"
+version = v"10.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://ftp.gnu.org/gnu/gdb/gdb-10.1.tar.xz", "f82f1eceeec14a3afa2de8d9b0d3c91d5a3820e23e0a01bbb70ef9f0276b62c0")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+apk add texinfo
+cd $WORKSPACE/srcdir/gdb-10.1/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} 
+make -j${nproc} all
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
```

Comment:
Maybe it'll work for other Musl platforms?

Reply:
no, it doesn't - I tried them all.

Reply:
Interesting :confused:

Reply:
There are some patches in Alpine Linux: https://git.alpinelinux.org/aports/tree/main/gdb?h=master.